### PR TITLE
Windows environment variables

### LINKED ISSUE #60: `System.getenv()` variables are always passed to a process on Windows
`System.getenv()` variables (not overwritten ones) are always passed to a process on Windows. The root cause is this line: https://github.com/brettwooldridge/NuProcess/blob/master/src/main/java/com/zaxxer/nuprocess/windows/WindowsProcess.java#L644

### PR DESCRIPTION
**Pull request by ilya-klyuchnikov**
fixes #60 
Tests:
https://ci.appveyor.com/project/ilya-klyuchnikov/nuprocess/build/13 (before fix)
https://ci.appveyor.com/project/ilya-klyuchnikov/nuprocess/build/14 (after fix)

### CODE REVIEW

File: src/test/java/com/zaxxer/nuprocess/EnvironmentTest.java [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+		} else {
+			command = new String[] { "env" };
+		}
+	}
+
+	@Test
+	public void emptyEnv() throws InterruptedException, IOException 
+	{
+		Map<String, String> env = new HashMap<String, String>();
+		if (isWin) {
```
Comment (ilya-klyuchnikov) [inline src/test/java/com/zaxxer/nuprocess/EnvironmentTest.java]: Java always add `SystemRoot` to the environment on Windows

Comment (brettwooldridge) [inline src/test/java/com/zaxxer/nuprocess/EnvironmentTest.java]: If Java adds ``SystemRoot``, shouldn't NuProcess?

Comment (brettwooldridge) [inline src/test/java/com/zaxxer/nuprocess/EnvironmentTest.java]: Bump.

Comment (brettwooldridge) [inline src/test/java/com/zaxxer/nuprocess/EnvironmentTest.java]: And update Re: my last question?

**Comment by ilya-klyuchnikov** [comment]
Thanks, for merging. However, I have one more addition - https://github.com/ilya-klyuchnikov/NuProcess/commit/b8f2f9fc78e82731271c28780935899e2b7a2880. The last commit is about handling case sensitivity in the same way as Java does in `java.lang.ProcessEnvironment` on Windows (Mac and Linux versions are different). The point is that when merging `System.environ()` with a user provided environment Java compares names up to case sensitivity.

**Comment by bturner** [comment]
I've opened pull request #103 to apply the changes @ilya-klyuchnikov suggested after this was merged, to fix the build on Windows.

**Pull request merged**